### PR TITLE
Feature/testing updates

### PR DESCRIPTION
**Pull request by makosblade**

### CODE REVIEW

File: src/main/java/com/kryptnostic/rhizome/hazelcast/serializers/RhizomeUtils.java [hunk 1]
```diff
@@ -101,7 +151,8 @@ public static void writeByteArray( ObjectOutput out, byte[] bytes ) throws IOExc
 
         public static String loadResourceToString( final String path ) {
             String resource = null;
-            try ( final InputStream stream = Thread.currentThread().getContextClassLoader().getResourceAsStream( path ) ) {
+            try ( final InputStream stream = Thread.currentThread().getContextClassLoader()
```
Comment (geekbeast) [inline src/main/java/com/kryptnostic/rhizome/hazelcast/serializers/RhizomeUtils.java]: Guava Resources does most of this already. Should probably update calls to use guava instead.

Comment (makosblade) [inline src/main/java/com/kryptnostic/rhizome/hazelcast/serializers/RhizomeUtils.java]: Yeah, I 100% don't understand why this code is around anymore :( Will update

File: src/main/java/com/kryptnostic/rhizome/mappers/KeyMapper.java [hunk 2]
```diff
@@ -1,7 +1,6 @@
 package com.kryptnostic.rhizome.mappers;
 
 public interface KeyMapper<K> {
-    public static final String ID_ATTRIBUTE = "id";
     public static final String DEFAULT_SEPARATOR = ":";
```
Comment (geekbeast) [inline src/main/java/com/kryptnostic/rhizome/mappers/KeyMapper.java]: Do we still use this anywhere?

Comment (makosblade) [inline src/main/java/com/kryptnostic/rhizome/mappers/KeyMapper.java]: Yep, a couple of the keymappers still use it.

Comment (geekbeast) [inline src/main/java/com/kryptnostic/rhizome/mappers/KeyMapper.java]: Bleh.

Comment (makosblade) [inline src/main/java/com/kryptnostic/rhizome/mappers/KeyMapper.java]: Yeah, Cassandra 3 with TypeCodecs will make it un-necessary

**Pull request merged**